src/Adapter/Alias/Repository/AliasRepository.php

```diff
@@ -151,4 +151,17 @@ public function getAliasesBySearchTerm(string $searchTerm): array
 
         return $qb->execute()->fetchFirstColumn();
     }
+
+    /**
+     * @param Alias $alias
+     * @param array $propertiesToUpdate
```

Comment:
define array type with psalm (you should be able to copy it from the partiallyUpdateObjectModel definition)